.circleci/config.yml

```diff
@@ -78,14 +76,20 @@ jobs:
 
             # Exclude multi-process transport tests because
             # they appear to have some crashing problem...
-            EXCLUDE="Transport/TransportMultiProcTest*:$EXCLUDE"
+            # EXCLUDE="Transport/TransportMultiProcTest*:$EXCLUDE"
 
             # Exclude memory leak test for tsan job (false positive).
-            if [ "$CIRCLE_JOB" == "tsan" ]; then
```

Comment:
is it true that this is not working before this change as it is not doing regex match?

Reply:
yes, it is, the original job has name "tsan"